platform-sdk/platform-apps/tests/AddressBookTestingTool/src/main/java/com/swirlds/demo/addressbook/AddressBookTestingToolState.java

```diff
@@ -233,26 +237,6 @@ private void handleTransaction(@NonNull final ConsensusTransaction transaction)
         runningSum += delta;
     }
 
-    /**
-     * {@inheritDoc}
-     */
-    @Override
```

Comment:
Same here. We may want to do here something similar to this commit https://github.com/hashgraph/hedera-services/pull/16352/commits/a53a4a064b2d487465415a132e46fbf985e268cc